flow-typed/components.js

```diff
@@ -10,3 +10,8 @@ declare type AddressPickerAddress = {|
   address: Address,
   title: string,
 |}
+
+declare type StyleComponent<Theme: ?string> = {
```

Comment:
We changed the name to Base Component

Reply:
And also maybe call it `Themes` instead of `Theme`? Because there could be multiple

Reply:
@stcherenkov https://github.com/jibrelnetwork/frontend/blob/master/codestyle.md#union-types

Reply:
Hm, well, good point